src/main/java/com/thoughtworks/pacman/ui/screens/SaveScreen.java

```diff
@@ -0,0 +1,102 @@
+package com.thoughtworks.pacman.ui.screens;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.*;
+import java.io.*;
+
+import com.thoughtworks.pacman.core.Game;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.nio.file.Paths;
+
+public class SaveScreen extends WindowAdapter implements ActionListener, ItemListener {
+    private JComboBox<String> comboBox;
+    private JPanel panel;
+    private JFrame frame;
+    private JLabel label;
+    private JButton button;
+    private String playerName;
+
+    private final Game game;
+
+    public SaveScreen(Game game){
+        this.game = game;
+        if(game != null) {
+
+            for (int i = 0; i < game.getGhosts().length; i++) {
+                game.getGhosts()[i].setSPEED(0);
+            }
+
+            frame = new JFrame("Save and Exit");
+            label = new JLabel("Select Player");
```

Comment:
Neden player seciyoruz?

Reply:
Hocam bunları issue açıklamamdaki "There should be options to determine the saving location" maddesinden dolayı yaptım aslında. Tam olarak nasıl bir seçenek koymam gerektiğine emin olamadım çünkü o yüzden farklı opsiyonlar seçtirdim. Burada ne yapmamı önerirsiniz bunun yerine?

Reply:
Anladım hocam aşağıda tanımlamışsınız teşekkür ederim.